jumpscale/packages/vdc/billing.py

```diff
@@ -0,0 +1,143 @@
+import os
+import requests
+from decimal import Decimal
+from jumpscale.loader import j
+from jumpscale.sals.vdc.size import VDC_SIZE
+from jumpscale.sals.vdc.models import KubernetesRole
+
+
+BASE_CAPACITY = int(os.getenv("BASE_CAPACITY", 14))
+
+
+def get_addons(flavor, kubernetes_addons):
+    """Get all the addons on the basic user plan
+
+    Args:
+        flavor(str): user flavor for the plan
+        kubernetes_addons(list): all kubernetes nodes 
+    
+    Returns:
+        addons(list): list of addons used by the user over the basic chosen plan
+
+    """
+    plan = VDC_SIZE.VDC_FLAVORS.get(flavor)
+    plan_nodes_count = plan.get("k8s").get("no_nodes")
+    plan_nodes_size = plan.get("k8s").get("size")
+    addons = list()
+    for addon in kubernetes_addons:
+        if addon.role != KubernetesRole.MASTER:
+            if addon.size == plan_nodes_size:
+                plan_nodes_count -= 1
+                if plan_nodes_count < 0:
+                    addons.append(addon)
+            else:
+                addons.append(addon)
+    return addons
+
+
+def get_prices():
+    """Get the prices of the addons and plans from github 
+    Returns:
+        VDC_PRICES(dict): return VDC_prices dictionary form out config
+    """
+    # Store prices from github
+    if not j.config.get("VDC_PRICES"):
+        prices = requests.get(
```

Comment:
this request can fail, it should be handled and use the default values if so

Reply:
For now, we are using the prices on GitHub tell we have a default prices